openshift-knative-operator/cmd/operator/kodata/knative-eventing/1.7/5-eventing-post-install.yaml

```diff
@@ -224,5 +224,5 @@ spec:
             runAsNonRoot: true
             capabilities:
               drop:
-              - ALL
+              - all
```

Comment:
This should be "ALL".

Reply:
yeah, I know, I thought our midstreams have all these fixes - let me double check

Reply:
https://github.com/openshift-knative/eventing/pull/58